datanode/networkhistory/snapshot/service_load_snapshot.go

```diff
@@ -272,15 +278,15 @@ func (b *Service) loadSnapshot(ctx context.Context, loadLog LoadLog, snapshotDat
 	}
 
 	loadLog.Infof("copying %s into database", snapshotData.UncompressedDataDir())
-
+	startTime := time.Now()
 	rowsCopied, err := b.copyDataIntoDatabase(ctx, vegaDbConn, decompressedFilesDestination,
 		filepath.Join(b.snapshotsCopyFromPath, snapshotData.UncompressedDataDir()), dbMetaData,
-		withIndexesAndOrderTriggers)
+		withIndexesAndOrderTriggers, historyTableLastTimestamps)
 	if err != nil {
 		return 0, fmt.Errorf("failed to copy uncompressed data into the database %s : %w", snapshotData.UncompressedDataDir(), err)
 	}
-
-	loadLog.Infof("copied %d rows from %s into database", rowsCopied, snapshotData.UncompressedDataDir())
+	elapsed := time.Since(startTime)
```

Comment:
Do we still need these time logs here, or was that part of the debugging/detective work?

Reply:
The elapsed time was added for the debugging, but I think it is nicer for the end user to see and if there are problems can be helpful so I kept it in.